plugin-manifest.ts

```diff
@@ -0,0 +1,26 @@
+import { EncodedExtension } from '@openshift/dynamic-plugin-sdk-webpack';
+import { ConsolePluginBuildMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack/lib/build-types';
+import { YAMLExposedModules, YAMLExtensions } from './src/templates/manifest';
+import {
+  FlagsExposedModules,
+  FlagsExtensions,
+} from './src/utils/flags/manifest';
+
+export const pluginMetadata: ConsolePluginBuildMetadata = {
+  name: 'networking-console-plugin',
+  version: '0.0.1',
+  displayName: 'Networking console plugin',
+  description: 'Plugin responsable for all the networking section ui code',
```

Comment:
Shouldn't we better use "responsible" with "i" for the description here? Google translator says "responsable" is a spanish word :D